pkg/meta/tkv_tikv.go

```diff
@@ -184,8 +184,25 @@ func (c *tikvClient) shouldRetry(err error) bool {
 	return strings.Contains(err.Error(), "write conflict") || strings.Contains(err.Error(), "TxnLockNotFound")
 }
 
-func (c *tikvClient) txn(f func(*kvTxn) error, retry int) (err error) {
-	tx, err := c.client.Begin()
+func (c *tikvClient) config(key string) interface{} {
+	if key == "startTS" {
+		ts, err := c.client.CurrentTimestamp(oracle.GlobalTxnScope)
+		if err != nil {
+			logger.Warnf("TiKV get startTS: %s", err)
```

Comment:
This result will inconsistent, return error

Reply:
fixed, return error when tikv get ts error.

Reply:
it's not fixed in last commit